devices/lidl.js

```diff
@@ -216,4 +217,16 @@ module.exports = [
             device.getEndpoint(1).saveClusterAttributeKeyValue('lightingColorCtrl', {colorCapabilities: 29});
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_htnnfasr'}],
+        model: 'PSBZS A1',
+        vendor: 'Parkside',
+        description: 'Smart Watering Timer',
+        supports: 'switch, timer',
+        fromZigbee: [fz.ignore_basic_report],
+        toZigbee: [tz.on_off, tz.lidl_watering_timer],
+        onEvent: tuya.onEventSetTime,
+        configure: async (device, coordinatorEndpoint, logger) => {},
+        exposes: [],
```

Comment:
I would expect `e.switch()` here.

Reply:
I haven't add it because the device is not reporting to coordinator, so after auto-off state in zigbee2mqtt is still ON. 
I'll add it anyway, just letting you know.

Reply:
done